schema.graphql

```diff
@@ -508,6 +508,7 @@ enum SortSearchInput {
 
 type Transporter {
```

Comment:
I was also considering to switch type name from `Transporter` to `Carrier`. 
Carrier is used across glossary https://kiwi.wiki/kiwi/handbook/#/comms/glossary and it will also be unified with new kiwi search schema.

What do you think?

Reply:
I think that is a good idea yes. We should use the names from the glossary

Reply:
+1

Reply:
Ok, thanks, switching this to WIP and will re-request reviews in a moment after renaming.

Reply:
Updated, renamed.